MANIFEST.in

```diff
@@ -1,7 +1,3 @@
-# Top-level Config
-include CHANGELOG
-prune tests
```

Comment:
packaging only includes `test` (not `testS`) by default: https://packaging.python.org/en/latest/guides/using-manifest-in/ we can put the changelog back (or not) in the docs dir PR